README.md

```diff
@@ -1 +1,13 @@
-# sis-extras-feedstock
\ No newline at end of file
+# sis-extras-feedstock
+
+home: https://github.com/blackary/sis-extras
+summary: Python package to make it easy to develop beautiful, performant streamlit-in-snowflake apps
+description: |
+  The sis-extras package provides a set of Python utilities designed
+  to help you develop beautiful, performant streamlit-in-snowflake
+  apps.
+license: MIT
+license_url: https://spdx.org/licenses/MIT.html
+license_family: MIT
+doc_url: https://github.com/blackary/sis-extras
+dev_url: https://github.com/blackary/sis-extras
```

Comment:
There should be empty lines between lines to make markdown format happy, see https://github.com/AnacondaRecipes/sis-extras-feedstock/blob/41421ccddfcf40fb04b7cc4a179c4e791d505111/README.md

Reply:
Hmm, that was automatically suggested/added.
I wonder if a list of items would be fine (which also can be auto-suggested).

Reply:
A list doesn't look as good as separate line.
(Neither looks great!)